application/common/components/Mailer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace common\components;
+
+use Aws\Ses\SesClient;
+use yii\mail\BaseMailer;
+
+class Mailer extends BaseMailer
```

Comment:
Would it be better to include "SES" (or "Ses") somewhere in this class's name or hierarchy, since it is specifically for sending email using SES? As-is, it seems unclear (to code that would use this) which implementation it is for, and as long as we support both SMTP and SES, it seems helpful for the class implementing each option to be clear about which option it is for.